src/misc/libraries/ConversionLib.sol

```diff
@@ -18,4 +19,37 @@ library ConversionLib {
 
         return price.mulUint256(MathLib.mulDiv(baseAmount, 10 ** quoteDecimals, 10 ** baseDecimals));
     }
+
+    /// @dev Converts asset amount to share amount.
+    function assetToShareAmount(
```

Comment:
NOTE: My plan is to merge this lib with `VaultPricingLib` in a follow-up PR. Or keep both libs and use `ConversionLib` in `VaultPricingLib`.